.github/workflows/smoke-tests.yml

```diff
@@ -45,8 +45,17 @@ jobs:
     - name: Python Checkstyle
       timeout-minutes: 20
       run: |
-        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903  ./src/*/*.py
-        # disabled warnings: import loading, todos, function members, duplicate code, public methods
+        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903 --disable=R0915 ./src/*/*.py
+        # disabled warnings: import loading, todos, function members, duplicate code, public methods, too many statements
+    - name: Python Unit Tests
+      timeout-minutes: 20
+      run: |
+        cd src/userservice
+        python3 -m venv env
+        source env/bin/activate
+        pip install -r tests/requirements-test.txt
+        python -m pytest -v -p no:warnings
+        deactivate
```

Comment:
"set -x" can be useful to see more detailed logs as the test is run

Reply:
I am running it with -v which is the verbose flag, not sure if set -x is necessary

Reply:
I was thinking of adding "set -x" as a bash command at the top, so we can see each command as they are run (like the License Check)

Reply:
done